src/SFML/Network/Ftp.cpp

```diff
@@ -30,15 +30,8 @@
 
 #include <SFML/System/Err.hpp>
 
-#include <algorithm>
```

Comment:
Are you sure you can remove this? There's a call to `std::copy` in this file.

Reply:
You're right, I missed a few lines. my bad 😕 
This means that "Algorithm" is already included indirectly in "SFML/Network/Ftp.hpp", "SFML/Network/IpAddress.hpp" or "SFML/System/Err.hpp", so deleting it will not result in any significant savings in compilation time.

I found two inclues in the "Unix" and "Win32" folders of the module Network: 
[Unix/SocketImpl.cpp](src/SFML/Network/Unix/SocketImpl.cpp) : the header `<ostream>`
[Win32/SocketImpl.cpp](src/SFML/Network/Win32/SocketImpl.cpp) : the header `<cstring>`

After examination, I found that they are not required.  What are your thoughts on this?